tools/lsp/common.rs

```diff
@@ -403,9 +401,24 @@ impl std::fmt::Debug for VersionedUrl {
 #[derive(Clone, Debug, serde::Deserialize, serde::Serialize, PartialEq, Eq)]
 pub struct Position {
     /// The file url
-    pub url: Url,
+    url: Url,
     /// The offset in the file pointed to by the `url`
-    pub offset: u32,
+    offset: u32,
```

Comment:
why not TextSize here?

Reply:
Because these are sent to the LSP and TextSize can not be serialized.